scarb/tests/build.rs

```diff
@@ -1112,6 +1114,129 @@ fn add_statements_functions_debug_info() {
     );
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct SourceCodeLocation {
+    pub line: usize,
+    pub col: usize,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct SourceCodeSpan {
+    pub start: SourceCodeLocation,
+    pub end: SourceCodeLocation,
+}
```

Comment:
Can't you import these?

Reply:
> This way the modification in the compiler will break the test, which imo is desired.

Can you elaborate please, what should be Scarb's team reaction to a change to this structure made on compilers side?

Reply:
@maciektr pinging on slack I presume 🤔 
It changes the scarb interface, even though the unstable one

Reply:
I am not sure if this kind of policing cairo-compiler changes is our job. It is defined in cairo repo instead of Scarb one for a reason (or at least it should be).

Reply:
I don't really expect you to actively manage that, just wanted to increase the chance it gets detected early. If you feels strong about this, I will use the compiler's structures.

Reply:
You can leave it as is :) 
Just wanted to discuss what steps do you expect.